views/mainMenu.ejs

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```

Comment:
Further to my point about not needing node - note how this is actually just a static HTML file - there's no reason you couldn't have served it as such and be done with node!